.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: ruby
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.6
+  - 2.1.10
```

Comment:
Yes, I've been thinking of adding ruby 2.0.x there too, but since we've got only three api keys, I've decided to go with ruby 2.1 and higher. Could you please give me a test key to use with ruby 2.0.x? We also have 2.4 already, so probably we'll need to have it as well; then I need two more test api keys.